README.md

```diff
@@ -11,6 +11,7 @@ This adds three magic helpers to use with Alpine JS. ***More to come!***
 | [`$component/$parent`](#component) | Natively access and update data from other components or the parent component. |
 | [`$fetch`](#fetch) | Using Axios, fetch JSON from an external source.  |
 | [`$interval`](#interval) | Run a function every n milliseconds. Optionally start and stop the timer. |
+| [`$scroll`](#scroll) | Scroll the page to a specific point (smoothly if supported by the browser). |
```

Comment:
Do you think we should support smooth scrolling regardless of native browser support? Personally I think all magic methods should work the same in all browsers so that users can reliably drop them into their projects and not even need to check whether it works everywhere. 

Especially since smooth scrolling is very common I think this would be a nice addition

Reply:
Yeah but only if it doesn't add 20KB :) The only black sheep are IE11 (and I'd rather people will stop using it) and Safari (which is as bad as IE because apple shouldn't really do software :D ).

Reply:
This one looks about 4kb

https://github.com/iamdustan/smoothscroll

Reply:
And I agree about Apple =/

Reply:
I wonder if we could load it in conditionally from a CDN only if it's needed.

Reply:
okay, we can support it adding a polyfill (iamdustan/smoothscroll). Scroll will increase up to 5kb (unzipped).
Let me know what you think

Reply:
I think it's a good tradeoff but let's see what others say. 

The designers I work with always remind me to support Safari when I implement a smooth scroll (usually a "back to top" button), so I personally wouldn't be able to use this otherwise.

Reply:
cool, I didn't refresh the page so I didn't see that you suggested the same polyfill and my reply sounded a bit silly, hahaha. I'm happy with that library, let's see what Hugo and Ryan think.

Reply:
I've also lost my phone in my house (baby brain , hahah, so I can't see telegram in case you messaged me there)

Reply:
I sent some messages about the `$inView` stuff, so you can check that whenever